src/index.html

```diff
@@ -4,9 +4,333 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="На главную">
+      </a>
+      <nav class="header__nav-bar">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link is-active">
+            Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+            href="#"
+            class="header__nav-link"
+            data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Гаджеты
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#"  class="header__nav-link">
+            Планшеты
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Фото
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">
+            Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-section">
```

Comment:
```suggestion
    </header>
    
    <main class="main-section">
```
Добавляй пустые линии между блоками